packages/fxa-settings/src/components/images/index.tsx

```diff
@@ -5,10 +5,9 @@ import { ReactComponent as RecoveryCodes } from './graphic_recovery_codes.svg';
 import { ReactComponent as TwoFactorAuth } from './graphic_two_factor_auth.svg';
 import { ReactComponent as PushFactorAuth } from './graphic_push_factor_auth.svg';
 import { ReactComponent as Mail } from './graphic_mail.svg';
-import { ReactComponent as SecurityShield } from './graphic_security_shield.svg';
-import { ReactComponent as Key } from './graphic_key.svg';
-import { ReactComponent as Lock } from './graphic_lock.svg';
-import { ReactComponent as Lightbulb } from './graphic_lightbulb.svg';
+import { ReactComponent as Key } from './graphic_recovery_key.min.svg';
```

Comment:
You're probably aware but depending on who gets their PR merged first ([link](https://github.com/mozilla/fxa/pull/17541/files#diff-e792549087b9465ea0c18bd9671dc25bf8432df919f4cd79965ccff745796f0aR13)) will have a smol merge conflict here.

Reply:
No worries! Truly tiny merge conflicts :)